From 2612676ef06967e0849ff1579cbceb2d22969516 Mon Sep 17 00:00:00 2001
From: S-trace <S-trace@list.ru>
Date: Sun, 17 Jan 2016 08:26:42 +0300
Subject: [PATCH 4/4] [RECOVERY] props: override ro.product.device

Let it be D10A_HighScreen
This is necessary to fix stock ROMs device check

Change-Id: I46bdf89f24868b05bef75044fa85a5c1ddf7e5fd
---
 core/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/Makefile b/core/Makefile
index f61b210..5577c5f 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1090,6 +1090,7 @@ define build-recoveryramdisk
   $(hide) cat $(INSTALLED_DEFAULT_PROP_TARGET) $(recovery_build_prop) \
           > $(TARGET_RECOVERY_ROOT_OUT)/default.prop
   $(hide) sed -i 's/ro.build.date.utc=.*/ro.build.date.utc=0/g' $(TARGET_RECOVERY_ROOT_OUT)/default.prop
+  $(hide) sed -i 's/ro.product.device=.*/ro.product.device=D10A_HighScreen/g' $(TARGET_RECOVERY_ROOT_OUT)/default.prop
   $(BOARD_RECOVERY_IMAGE_PREPARE)
   $(if $(filter true,$(BOARD_BUILD_SYSTEM_ROOT_IMAGE)), \
     $(hide) mkdir -p $(TARGET_RECOVERY_ROOT_OUT)/system_root; \
-- 
2.9.3

